priam/src/main/java/com/netflix/priam/IConfiguration.java

```diff
@@ -213,6 +213,27 @@
      */
     public String getHeapNewSize();
 
+    /**
+     * Cron expression to be used to schedule regular compactions.
+     *
+     * @return Compaction cron expression.
+     * @see <a href="http://www.quartz-scheduler.org/documentation/quartz-2.x/tutorials/crontrigger.html">quartz-scheduler</a>
+     * @see <a href="http://www.cronmaker.com">http://www.cronmaker.com</a> To build new cron timer
+     */
+    default String getCompactionCronExpression(){
+        return "-1";
+    }
+
+   /**
+    * Column Family(ies), comma delimited, to start compactions.
+    * Note:  the expected format is keyspace.cfname. If no value is provided then compaction is scheduled for all KS,CF(s)
+    *
+    * @return Column Family(ies), comma delimited, to start compactions.  If no filter is applied, returns null.
+    */
+    default String getCompactionCFList(){
```

Comment:
Do you want to consider exclude list and include list, so that on multi-tenant cluster, when there are 100's of CFs exists, it would be difficult to manage this list to exclude just few CFs.